synopsys-task/src/synopsys-task/application-constant.ts

```diff
@@ -206,16 +208,137 @@ export const BRIDGE_COVERITY_VERSION_KEY = "bridge_coverity_version";
 export const COVERITY_VERSION_KEY = "coverity_version";
 export const COVERITY_VERSION_KEY_CLASSIC_EDITOR = "coverityVersion";
 
-// Bridge Exit Codes
+// Bridge and ADO Exit Codes
 export const EXIT_CODE_MAP = new Map<string, string>([
-  ["0", "Bridge execution successfully completed"],
-  ["1", "Undefined error, check error logs"],
-  ["2", "Error from adapter end"],
-  ["3", "Failed to shutdown the bridge"],
-  ["8", "The config option bridge.break has been set to true"],
-  ["9", "Bridge initialization failed"],
+  [
+    ErrorCode.SUCCESSFULLY_COMPLETED.toString(),
+    "Bridge execution successfully completed",
+  ],
+  [
+    ErrorCode.UNDEFINED_ERROR_FROM_BRIDGE.toString(),
+    "Undefined error, check error logs",
+  ],
+  [ErrorCode.ADAPTER_ERROR.toString(), "Error from adapter end"],
+  [
+    ErrorCode.BRIDGE_SHUTDOWN_FAILURE.toString(),
+    "Failed to shutdown the bridge",
+  ],
+  [
+    ErrorCode.BRIDGE_BREAK_ENABLED.toString(),
+    "The config option bridge.break has been set to true",
+  ],
+  [
+    ErrorCode.BRIDGE_INITIALIZATION_FAILED.toString(),
+    "Bridge initialization failed",
+  ],
+  [
+    ErrorCode.MISSING_AT_LEAST_ONE_SCAN_TYPE.toString(),
```

Comment:
if you can add comment that ado related error code starting here that will be good